Add more secondary requirements (#1986)
act-rules/act-rules.github.io: _rules/aria-state-or-property-valid-value-6a7281.md

    failed: not satisfied
    passed: satisfied
    inapplicable: satisfied
  wcag20:1.3.1: # Info and Relationships (A)

>     secondary: true

Reviewer: Should we nonetheless have mapping for secondary requirements?
Notably, this match the definition of only satisfying one of the two conditions for Conformance requirements; and for the "strict version" it would emphasise that the rule does also test for these failures.

I feel at least we should have the `title` part, and I think we should also have the outcomes mapping.

That depends also a bit how we intend to show them in the rule page 🤔

Author: The `title` isn't necessary for WCAG SCs. The other part is a discussion for the rules format. Right now we didn't put this as required.

Reviewer: Ah, true, the title is added automatically for WCAG SCs 👍 

For the mapping, even if not required, it may make sense to have it. Granted, maybe not in all cases 🤔 I feel it would be helpful for, e.g., "color contrast minimum" to also show the mapping to 1.4.6.

## Background

> Using invalid ARIA attribute values is often the result of a typo or other developer error. These attributes are then either ignored, or a default value is assumed by browsers and assistive technologies. This often means that a state or property should exist does not. This can cause issues under [success criterion 1.3.1 Info and Relationships][sc131] or [4.1.2 Name, Rule Value][sc412].

Author: ```suggestion
Using invalid ARIA attribute values is often the result of a typo or other developer error. These attributes are then either ignored, or a default value is assumed by browsers and assistive technologies. This often means that a state or property which should exist is missing. This can cause issues under [success criterion 1.3.1 Info and Relationships][sc131] or [4.1.2 Name, Rule Value][sc412].
```

> Using invalid ARIA attribute values is often the result of a typo or other developer error. These attributes are then either ignored, or a default value is assumed by browsers and assistive technologies. This often means that a state or property which should exist is missing. This can cause issues under [success criterion 1.3.1 Info and Relationships][sc131] or [4.1.2 Name, Rule Value][sc412].

Reviewer 2: ```suggestion
Using invalid ARIA attribute values is often the result of a typo or other developer error. These attributes are then either ignored, or a default value is assumed by browsers and assistive technologies. This often means that a state or property which should exist is missing or has an unexpected value. This can cause issues under [success criterion 1.3.1 Info and Relationships][sc131] or [4.1.2 Name, Rule Value][sc412].
```
